config/clusterPipeline.json

```diff
@@ -0,0 +1,5 @@
+{
+    "sinks":
```

Comment:
I think we've been standardizing on two spaces instead of four. 

So what is the plan here for configuration? RPM is installed and then you need to overwrite the configuration files? I don't think that's a bad answer for rpm, but just wanted to check.